src/io/flutter/settings/FlutterSettings.java

```diff
@@ -269,11 +270,16 @@ public boolean isEnableHotUiInCodeEditor() {
   }
 
   public boolean isEnableEmbeddedBrowsers() {
-    return getPropertiesComponent().getBoolean(enableEmbeddedBrowsersKey, true);
+    // TODO(helinx): Change default value once JxBrowser is compatible with MacOS 11.
+    return getPropertiesComponent().getBoolean(enableEmbeddedBrowsersKey, !SystemInfo.isMacOSBigSur);
   }
 
   public void setEnableEmbeddedBrowsers(boolean value) {
-    getPropertiesComponent().setValue(enableEmbeddedBrowsersKey, value, true);
+    // TODO(helinx): Remove this and change default value below once JxBrowser is compatible with MacOS 11.
+    if (SystemInfo.isMacOSBigSur) {
```

Comment:
I don't think we'll need to change the behavior of the setEnableEmbeddedBrowsers() w/ the change to isEnableEmbeddedBrowsers() I suggested above.

Reply:
I thought the setting would show up as checked but disabled, but it's always empty if disabled.